src/commands/applications/list.ts

```diff
@@ -134,7 +150,7 @@ use applications:get to get full details for a single app.
     }
 
     if (totalSize > count) {
-      this.log(`\nShowing ${chalk.cyan(count)} of ${chalk.cyan(totalSize)} application${s(count)}.\n`)
+      this.log(`\nShowing ${chalk.cyan(count)} of ${chalk.cyan(totalSize)} application${s(totalSize)}.\n`)
```

Comment:
Need to apply the treatment discussed for projects:list here and move message to after the list is displayed.